item.rb

```diff
@@ -0,0 +1,17 @@
+class Item
+  attr_accessor :genre, :source, :author, :label
```

Comment:
- [ ] Please, ensure to add the methods to access and modify data from the publish_date and archived variables.